### DISCUSSION TITLE
openssl is required even for system python

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#1620] [Q&A]
使用系统python binary/libx11等时，仍然提示安装openssl：
![2021-08-31_13-49](https://user-images.githubusercontent.com/42310255/131449087-b0efd04f-7a81-46ae-a3cf-a64ef3e45926.png)
但系统库实际并不需要openssl。如何改进？

**Comment by waruqi** [comment]
有个 private，肯定有地方 add_deps ，将它作为 private 引入了，没走系统库。。

magnum -> libsdl -> package:add("deps", "libxext", {private = true}) ?

https://github.com/xmake-io/xmake-repo/blob/bdff6bcebe91ccd1ec4ca195835867724f90251d/packages/l/libsdl/xmake.lua#L58
